docs/maintenance/ISSUES.md

```diff
@@ -0,0 +1,101 @@
+---
+id: issues
+sidebar_label: Issue Management
+title: Issue Management
+---
+
+This document serves as a guide for how you might manage issues, also known as issue triaging.
+
+Use your best judgement when triaging issues, but most of all remember to be **kind, friendly, and encouraging** when responding to users.
+
+:::tip
+If you're ever unsure on any part of issue management, don't hesitate to loop in a maintainer that has more context to help!
+:::
+
+## Issue Flow
+
+[Issues pending triage](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3Atriage) are given the `triage` label.
+Most issues go through the following triage flow when created:
+
+1. Some user files the issue, and it is given the `triage` label automatically
+2. A maintainer will ensure the issue is valid:
+   - If it doesn't fill out an appropriate template with enough information:
+     - Add the `please fill out the template` and `awaiting response` labels
+     - Ask the poster for more information using a _Needs More Info_ response
```

Comment:
This page should start with a mention of the saved responses, or link these to the section below